### DISCUSSION TITLE
Evaluation of AUC in multi-sequencing run experiment

### DISCUSSION THREAD

**Discussion by DominicShayler** [aertslab/SCENIC#317] [* General SCENIC questions]
I'm working with a final dataset that is the result of several rounds of sample collection and sequencing. For my standard analysis, Seurat has proven invaluable in normalizing and visualizing in UMAP. However I know that SCENIC takes raw counts as input, and that AUC scores are stated to only be comparable for a single experiment. Does this mean that even for a single regulon, AUC values are not going to be comparable due to the multi-seq nature of the total dataset? If so, does anyone have any suggestions for tackling this? 

I've been looking at the z-score outputs demonstrated in the tutorial analysis, however z-score transformation seems to create some odd patterns of regulon activity I wouldn't expect.

**Comment by lucygarner** [comment]
I have posted a similar question relating to dealing with batch effects using pySCENIC. If you don't see batch effects in your UMAP, then you can go ahead and run pySCENIC on raw counts from the entire dataset. However, if there are clear batch effects between experiments, then this is a bit trickier - I am in this position and haven't yet found a good solution.
